lib/engine/assignable.rb

```diff
@@ -2,6 +2,8 @@
 
 module Engine
   module Assignable
+    ASSIGNED = 1
+    STACKABLE = 2
```

Comment:
In Ruby, you generally use symbols instead of defining constants. `assignments[key] = :stackable`

Reply:
Change for a generic stack_group: so that we can support multiple stack_groups

Reply:
Removed